package.json

```diff
@@ -174,7 +172,6 @@
     "sass-true": "6.0.1",
     "sinon": "12.0.1",
     "snyk": "1.817.0",
-    "storybook-addon-html-document": "1.0.1",
```

Comment:
This is never loaded in [main.js](https://github.com/uswds/uswds/blob/develop/.storybook/main.js#L37). Storybook-addon-html-document was also flagged in this high severity [Dependabot alert](https://github.com/uswds/uswds/security/dependabot/34)